docs/rules/valid-prop-names-in-kit-pages.md

```diff
@@ -53,7 +53,7 @@ At SvelteKit v1.0.0-next.405, instead of having multiple props corresponding to
 
 ## :wrench: Options
 
-Nothing. But if use are using not default routes folder, please set configuration according to the [user guide](../user-guide.md#settings-kit).
+Nothing. But if use are using not default routes folder, please set configuration according to the [user guide](../user-guide.md#settings-svelte-kit).
```

Comment:
oops sorry!

Reply:
I couldn't reproduce the build error locally 😓, but it was warned in the workflow, so I changed it.